mtp_api/apps/transaction/views.py

```diff
@@ -74,8 +74,10 @@ def patch_processed(self, request, *args, **kwargs):
         try:
             return self.partial_update(request, *args, **kwargs)
         except InvalidCreditStateException as e:
-            logger.warning('Some transactions failed to update: [%s]' %
-                           ', '.join(map(str, e.conflict_ids)))
+            logger.warning(
+                'Some transactions failed to update',
+                {'transaction_id_list': e.conflict_ids}
```

Comment:
I'm curious, this list is going to be serialised into a string like `[1, 2, 3]` on Sentry right? Or maybe it just stays a list there?

Reply:
I doubt Sentry will transform the data type, but I haven't validated that